packages/react-slider/src/components/Slider/useSliderState.tsx

```diff
@@ -211,7 +199,7 @@ export const useSliderState = (state: Pick<SliderState, keyof SliderCommon | key
   state.trackWrapper.children = null;
 
   // Track Props
-  state.track.className = 'ms-Slider-track';
+  state.track.className = state.track.className || 'ms-Slider-track';
```

Comment:
These should probably be concatenated rather than using one or the other.

Though actually come to think of it...do we have a standard in converged for whether to include these `ms-Whatever` class names or not? And if we do want to include them, does `make-styles` support merging in class names (so this could be moved to `useSliderStyles`)?

Reply:
I agree that we should probably concatenate the `classNames` here, we also do have standard support for merging classNames. I think I agree with @ecraig12345 when she says we should move this to `useSliderStyles`.

Reply:
I think the conclusion based on the slider sync yesterday was that we should remove the `ms-Slider` class names for now?

Reply:
I removed the classNames from both the track and thumb. The design no longer has them changing color in sync during focus so a className selector is no longer needed for styling.